hr_time/public/js/easy_checkin_dialog.js

```diff
@@ -20,58 +20,110 @@ export class EasyCheckinDialog {
     }
 
     /**
-     * Shows the checkin dialog
+     * Checks if the employee has worklogs today before showing the dialog
      */
     show() {
         let checkin_dialog = this;
 
-        let dialog = new frappe.ui.Dialog({
-            title: __("Checkin"),
-            fields: [
-                {
-                    label: 'Action',
-                    fieldname: 'action',
-                    fieldtype: 'Select',
-                    options: this.options,
-                    default: this.default
+        // First, fetch the employee_id from the backend
+        frappe.call({
+            method: "hr_time.api.employee.api.get_current_employee_id",  // Server-side call to fetch employee_id
+            callback: function(response) {
+                const employee_id = response.message;  // The employee_id returned from the server
+
+                if (!employee_id) {
+                    frappe.throw(__('No employee ID found for the current user.'));
+                    return;
                 }
-            ],
-            size: 'small', // small, large, extra-large
-            primary_action_label: __("Submit", undefined, "checkin"),
-            primary_action(values) {
-                frappe.call({
-                    method: "hr_time.api.flextime.api.submit_easy_checkin",
-                    args: {
-                        action: values.action
-                    },
-                    callback: (response) => {
-                        checkin_dialog.refresh_dashboard()
-                        EasyCheckinStatus.render();
-                        checkin_dialog.preload();
-
-                        let message = "Successfully checked in";
-
-                        switch (values.action) {
-                            case "Break":
-                                message = "Successfully checked out for break";
-                                break;
-                            case "End of work":
-                                message = "Successfully checked out for end of work"
-                                break;
-                        }
 
-                        frappe.show_alert({
-                            message: __(message),
-                            indicator: 'green'
-                        }, 5);
+                // Create the dialog only after the employee_id is fetched
+                let dialog = new frappe.ui.Dialog({
+                    title: __("Checkin"),
+                    fields: [
+                        {
+                            label: 'Action',
+                            fieldname: 'action',
+                            fieldtype: 'Select',
+                            options: checkin_dialog.options,
+                            default: checkin_dialog.default
+                        },
+                        {
+                            fieldtype: 'Button',
```

Comment:
Instead of a button, please add a message box, which either is green or red with an additional icon indicator (checkmark or cross for example).

- [ ] The message box should include a button for creating a worklog (but only if no worklog was found so 
- [ ] The message box should only be visible if the employee had been already checked in today.
- [ ] Add an additinal icon indicator (e.g. checkmark or cross)
- [ ] Message box should be translated to German